meta-resin-common/recipes-support/resin-expand/resin-data-expander.bb

```diff
@@ -1,35 +0,0 @@
-DESCRIPTION = "Resin data partition expander"
```

Comment:
I suggested to rephrase it like:

"we mount machine-id in initramfs which is on the resin-state logical partition in the extended primary partition)."

Currently it is like:

"we mount machine-id in initramfs which is on state logical partition in the extended primary partition)."

So it lacks a "the" and it is also just "state partition" instead of "resin-state partition"